core/math/a_star.cpp

```diff
@@ -438,6 +449,18 @@ PoolVector<int> AStar::get_id_path(int p_from_id, int p_to_id) {
 	return path;
 }
 
+void AStar::enable_point(int p_id) {
```

Comment:
I would honestly just do
AStar::set_point_disabled(id,bool)
and
AStar::is_point_disabled(id)

Because the default behavior is that this is all enabled, and disabled is done manually.

Reply:
Done !